Update writing_training.md (#2716)
nf-core/website: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/writing_training.md

#### Use this as a template.

Use this file `https://github.com/nf-core/website/blob/main/sites/docs/src/content/docs/tutorials/contributing_to_nf-core/writing_training.md` as a template to guide you.

1. Make a fork of the [nf-core website repository](https://github.com/nf-core/website/).

> 2. Make a new branch from main with `git checkout -b main` then branch off your fork. Name it after the guide you intend to submit. For example: `git checkout -b gitpod_training_guide`

Reviewer: I think `git checkout main` would try to create a new branch called `main`, which we don't want..

```suggestion
2. Make sure you have the `main` branch checked out with `git checkout main` then branch off your fork. Name it after the guide you intend to submit. For example: `git checkout -b gitpod_training_guide`
```
